examples/19_DFSM/README.md

```diff
@@ -0,0 +1,21 @@
+This example introduces the basic workings of the derivative function surrogate modeling (DFSM) approach, and demonstrates a usecase for closed-loop simulations.
+
+Checkout `run_simulation.py` on how to use the DFSM with ROSCO for closed-loop simulation.
+
+The DFSM is available in the `dfsm_1p6.pkl` file. This model has been specifically built for simulating load cases from DLC 1.6.
```

Comment:
How do we generate this pickle file?  What's in the pickle file?  Can it be expressed in a readable way?